source/includes/extracts-query-bar.yaml

```diff
@@ -4,14 +4,40 @@ content: |
    
    For details on sampling, see the :ref:`FAQ <compass-faq-sampling>`. 
 ---
-ref: query-bar-type
+ref: query-bar-type-schema
 content: |
    You can type MongoDB queries into the query bar. In the query
    expression, you can use the various MongoDB :manual:`query operators
    </reference/operator/query/>` except for the :query:`$text` query
    operator.
 
-   As you type, the :guilabel:`Apply` button is disabled and the
+   As you type, the :guilabel:`Analyze` button is disabled and the
+   :guilabel:`Filter` label turns red until a valid query is entered.
+   Filter syntax requires strict :manual:`extended JSON
+   </reference/mongodb-extended-json/>` types. For details on using the
+   query bar, see :ref:`querying`.
+---
+ref: query-bar-type-documents
+content: |
+   You can type MongoDB queries into the query bar. In the query
+   expression, you can use the various MongoDB :manual:`query operators
+   </reference/operator/query/>` except for the :query:`$text` query
+   operator.
+
+   As you type, the :guilabel:`Find` button is disabled and the
+   :guilabel:`Filter` label turns red until a valid query is entered.
+   Filter syntax requires strict :manual:`extended JSON
+   </reference/mongodb-extended-json/>` types. For details on using the
+   query bar, see :ref:`querying`.
+---
+ref: query-bar-type-queryplan
+content: |
+   You can type MongoDB queries into the query bar. In the query
+   expression, you can use the various MongoDB :manual:`query operators
+   </reference/operator/query/>` except for the :query:`$text` query
+   operator.
+
+   As you type, the :guilabel:`Explain` button is disabled and the
    :guilabel:`Filter` label turns red until a valid query is entered.
```

Comment:
Excerpt from earlier response: "Note that these two actions occur simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential value."